### ISSUE TITLE
From [here](https://matheo.uliege.be/bitstream/2268.2/6801/5/s123578Jemine2019.pdf#page=12):

### ISSUE THREAD

**Issue by chaiyawatmixx** [CorentinJ/Real-Time-Voice-Cloning#242]
From [here](https://matheo.uliege.be/bitstream/2268.2/6801/5/s123578Jemine2019.pdf#page=12):

> A particularity of the SV2TTS framework is that all models can be trained
separately and on distinct datasets. For the encoder, one seeks to have a model
that is robust to noise and able to capture the many characteristics of the human
voice. Therefore, a large corpus of many different speakers would be preferable to
train the encoder, without any strong requirement on the noise level of the audios.
Additionally, the encoder is trained with the GE2E loss which requires no labels other
than the speaker identity. (...) For the datasets of the synthesizer and the vocoder, 
transcripts are required and the quality of the generated audio can only be as good 
as that of the data. Higher quality and annotated datasets are thus required, which 
often means they are smaller in size.

You'll need two datasets:
![image](https://user-images.githubusercontent.com/12038136/60648823-0c6c5580-9e41-11e9-9b3a-737379481824.png)

The first one should be a large dataset of untranscribed audio that can be noisy. Think thousands of speakers and thousands of hours. You can get away with a smaller one if you finetune the pretrained speaker encoder. Put maybe `1e-5` as learning rate. I'd recommend 500 speakers at the very least for finetuning. A good source for datasets of other languages is [M-AILABS](https://www.caito.de/2019/01/the-m-ailabs-speech-dataset/).

The second one needs audio transcripts and high quality audio. Here, finetuning won't be as effective as for the encoder, but you can get away with less data (300-500 hours). You will likely not have the alignments for that dataset, so you'll have to adapt the preprocessing procedure of the synthesizer to not split audio on silences. See the code and you'll understand what I mean.

Don't start training the encoder if you don't have a dataset for the synthesizer/vocoder, you won't be able to do anything then.

_Originally posted by @CorentinJ in https://github.com/CorentinJ/Real-Time-Voice-Cloning/issues/30#issuecomment-508381648_

**Comment by blue-fish** [comment]
Someone clicked "reference in new issue" on https://github.com/CorentinJ/Real-Time-Voice-Cloning/issues/30#issuecomment-508381648

There is nothing actionable here so this issue is being closed.
